spec/controllers/state_file/questions/verification_code_controller_spec.rb

```diff
@@ -119,5 +119,19 @@
         expect(response).to redirect_to(questions_code_verified_path)
       end
     end
+
+    context "with an invalid form" do
+      let(:intake) { create(:state_file_az_intake, contact_preference: "email", email_address: "someone@example.com", visitor_id: "v1s1t1n9") }
+
+      it "renders the edit page" do
+        post :update, params: { state_file_verification_code_form: { verification_code: "invalid" }}
+        expect(response).to render_template(:edit)
```

Comment:
i think both: 
```
expect(response).to render_template(:edit)
expect(assigns(:contact_info)).to eq "someone@example.com”
```
could be set in the same test set up? but i’m not totally sure why they’re different?